fdbclient/ServerKnobs.cpp

```diff
@@ -802,6 +802,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( EMPTY_READ_PENALTY,                                   20 ); // 20 bytes
 	init( DD_SHARD_COMPARE_LIMIT,                               1000 );
 	init( READ_SAMPLING_ENABLED,                                true ); if ( randomize && BUGGIFY ) READ_SAMPLING_ENABLED = false;// enable/disable read sampling
+	init( DD_TRACE_MOVE_BYTES_AVERAGE_INTERVAL,                   120);
+	init( MOVING_WINDOW_SAMPLE_SIZE,                   		 10000000); // 10MB
```

Comment:
```suggestion
	init( MOVING_WINDOW_SAMPLE_SIZE,                   	     10000000); // 10MB
```

Reply:
Thanks for pointing out!
That's very interesting because it was a tab(which was 8 spaces on Github) before `init`. Inside parenthesis, however, they were all spaces.
In our codes, I found there are lots of places where we use both tab and spaces together. E.g:
```
		DebugRelocationTraceEvent("ReceivedRelocateShard", distributorId)
		    .detail("KeyBegin", rd.keys.begin)
		    .detail("KeyEnd", rd.keys.end)
		    .detail("Priority", rd.priority)
		    .detail("AffectedRanges", affectedQueuedItems.size());
```
Maybe we could somehow fix it so that Github won't complain about it?(Also we could Hide whitespace in diff review, but I guess it's not an ideal solution...)